asm/internal/ll.bnf

```diff
@@ -1610,7 +1617,7 @@ UnnamedAddr
 
 OptCommaAlign
 	: empty
-	| "," Align
+	| "," Align CommaAttachedMDs
```

Comment:
Same as with https://github.com/llir/llvm/pull/21/files#r115116240

`OptCommaAlign` is not an instruction, and if possible the attached metadata should be added to the production rule of the instruction, specifically for optional alignment, the `load`, `store` and `alloca` instructions.

If you wish, we could merge it as is, and I'll look into moving the attached metadata to the relevant instructions?